docs/specification.md

```diff
@@ -341,28 +346,23 @@ Here is a snippet of sample Go code to illustrate:
   }
 ```
 
-When the client receives this signal, it SHOULD follow the recommendations
-outlined in documentation for
-[RetryInfo](https://github.com/googleapis/googleapis/blob/6a8c7914d1b79bd832b5157a09a9332e8cbd16d4/google/rpc/error_details.proto#L40):
+Here is a snippet of sample Go code to illustrate using `grpc-retry-pushback-ms`:
 
+```go
+// Do this on the server side.
+trailer := metadata.Pairs("grpc-retry-pushback-ms", "5000")
```

Comment:
Is this expected to be done _only_ if the server wants to pushback? It is not clear from the sample if this is some sort of permanent option that the server must always set or only set when the server is overloaded and needs to pushback the clients.